### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by juampynr**
This is a work in progress. There is some logic that we should change in order to make it testable but I think that I can get to 70% coverage without changing anything.

Here is the current report:

image

I will push a few commits more later.

### CODE REVIEW

**Comment by juampynr** [comment]
@nignatov I have added a Travis file to run tests. Can you give access to Travis on this repository at https://travis-ci.org/ so tests get executed on each pull request?

**Comment by juampynr** [comment]
50%

image

**Comment by juampynr** [comment]
60%

image

**Comment by juampynr** [comment]
72%.

image

@nignatov, thoughts on this?

File: composer.json [hunk 1]
```diff
@@ -26,7 +26,15 @@
       "nodespark\\DESConnector\\": "src/DESConnector/"
     }
   },
+  "autoload-dev": {
+    "psr-4": {
+      "nodespark\\DESConnector\Test\\": "tests/"
```
Comment (nignatov) [inline composer.json:31]: Shouldn't we have "\\\\Test" instead of "\Test"

**Comment by nignatov** [comment]
@juampynr otherwise is great you have written the tests. I've enabled the travis for this repo.

**Comment by nignatov** [comment]
I will merge this to test the travis-ci and we can fix the namespace after that.

**Pull request merged**